.github/workflows/ci.yml

```
@@ -0,0 +1,53 @@
+name: ci
+
+on: pull_request
+
+jobs:
+
+  frontend:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 10.x
+
+      - name: Install Node dependencies
+        run: |
+          npm install -g gulp-cli
+          npm config set package-lock false
+          npm install
+
+      - name: Lint front-end code
+        run: gulp lint
```

Our current best practice is not to install Gulp globally, but to use `npx` (or `yarn`) to invoke the local-to-the-project installation of Gulp.

```suggestion
      - name: Install Node dependencies
        run: |
          npm config set package-lock false
          npm install

      - name: Lint front-end code
        run: npx gulp lint
```

---

Made change as requested to follow best practice